sdcm/nemesis.py

```diff
@@ -3706,9 +3706,14 @@ def decommission_post_action():
             disrupt_func_kwargs={"target_node": self.target_node, "hard": True, "verify_ssh": True},
             delay=0
         )
-        timeout = 1200
-        if self.cluster.params.get('cluster_backend') == 'azure':
-            timeout += 1200  # Azure reboot can take up to 20min to initiate
+
+        if terminate_pattern in ["became a group 0 non-voter", "leaving token ring", "left token ring",
+                                 "Finished token ring movement"]:
+            timeout = MAX_TIME_WAIT_FOR_DECOMMISSION
+        else:
```

Comment:
I don't like that explicit log patterns in code, but as workaround while i don't push fixes is LGTM